### PR TITLE
PyGame

### LINKED ISSUE #290: Add PyGame
### Some reference to the engine, such as the developer site or a wikipedia article.

https://www.pygame.org/

### SteamDB links

https://steamdb.info/app/951360/
https://steamdb.info/app/550900/
https://steamdb.info/app/442210/

### Possible way to detect it

not sure

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
Fine. I'll do it myself.
closes #290

### SteamDB app page links to a few games using this
https://steamdb.info/app/442210/
https://steamdb.info/app/550900/
https://steamdb.info/app/660340/
https://steamdb.info/app/918170/
https://steamdb.info/app/951360/
https://steamdb.info/app/1364570/
https://steamdb.info/app/1604400/
https://steamdb.info/app/2213660/
https://steamdb.info/app/2336660/
https://steamdb.info/app/2363270/
https://steamdb.info/app/2389490/

### Brief explanation of the change
Find PyGame by it's directories and base file.

### CODE REVIEW

**Comment by MichaelHinrichs** [comment]
Found another one.
https://steamdb.info/sub/93793/

**Comment by MichaelHinrichs** [comment]
Pull my finger.

File: descriptions/Engine.PyGame.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[**PyGame**](https://www.pygame.org/) is a set of highly portable Python libraries designed for writing video games. It runs on almost all OSs and platforms. Pygame is released under the LGPL licence, and is completely free.
```
Comment (Lifeismana) [inline descriptions/Engine.PyGame.md:1]:
```suggestion
[**PyGame**](https://www.pygame.org/) is a set of Python modules designed for writing video games.
```

**Review by Lifeismana — CHANGES_REQUESTED** [review]
needs some non matching test and the generate strings script to be run

**Pull request merged**